src/olympia/constants/site_permissions.py

```diff
@@ -0,0 +1,4 @@
+# List of site_permissions names we know about.
+SITE_PERMISSIONS = {'midi-sysex'}  # WebMIDI
```

Comment:
nit: comment says "list" but it's defined as a set

Reply:
mmm now that I don't generate descriptions (Shane told me that wasn't necessary as about:addons will display the hosts/permissions in a better way) I don't need this until I start working on the form to validate the permissions we're asked to generate. So I'll remove for now.